test/In.ProjectEKA.HipServiceTest/DataFlow/Builder/TestBuilder.cs

```diff
@@ -0,0 +1,23 @@
+using In.ProjectEKA.HipService.DataFlow;
+
+namespace In.ProjectEKA.HipServiceTest.DataFlow.Builder
+{
+    using Bogus;
+
+    public class TestBuilder
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4941971)